infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -417,6 +430,11 @@ public boolean isServerReadyToHandleMsg(CorfuMsg msg) {
         return getState() == ServerState.READY;
     }
 
+    @Override
+    public boolean isServerReadyToHandleMsg(RequestMsg request) {
+        return getState() == ServerState.READY;
+    }
```

Comment:
Move this to the AbstractServer class all servers have the same logic (except the `SequencerServer)`

Reply:
Done.